### ISSUE TITLE
[bug] "The issuer certificate of a locally looked up certificate could not be found" error during initial sync

### ISSUE THREAD

**Issue by wokawoka** [d1vanov/quentier#416]
Hello guys, two days ago I installed quentier and launched the first sync of my notes soon hitting the API limit.
Waiting for several cool-downs, through a consecutive series of syncs I arrived at synchronising about 4700 notes out of about 17000 total, but at this point the synchronisation no longer proceeds and every time I launch the program, at the end of a brief phase of "Downloading chunks" I receive the following error each time:
```
2024-04-03 16:05:59.548 CEST libquentier/libquentier/src/synchronization/NoteStore.cpp:428 [Warn]: QEverCloud exception: SSL Errors:
The issuer certificate of a locally looked up certificate could not be found
```
Do you have any suggestions? 
Thanks!

OS - Linux Ubuntu 21.10
Quentier 0.5.0 master appimage downloaded from this github repo

**Comment by d1vanov** [comment]
This error comes from OpenSSL. Did you update some related package on your system recently?
Anyway, I'd recomment installing a native package from [launchpad](https://launchpad.net/~d1vanov/+archive/ubuntu/quentier-stable?field.series_filter=impish). AppImage format has lots of known problems and limitations, I consider it mostly a way to get familiar with the app but not a production grade solution.

**Comment by d1vanov** [comment]
Also, you probably won't be able to synchronize all your notes with current versions of Quentier. The problem is that special kind of sync logic is required to prevent it from trying to download the same data on each sync attempt - it has become apparent in #391. I sort of implemented that logic in #d1vanov/libquentier#206 along with new local storage (#d1vanov/libquentier#181) and now I'm trying to adapt the entire codebase to these changes. I can't tell when it'd be ready but hopefully, if things don't fall apart completely during the testing phase, I hope to release something this summer.
See also #413.

**Comment by wokawoka** [comment]
Thank you for the superquick answer.
I installed the deb version and now I'm no more receiving the error above but the with this version the syncing goes into timeout anyway.
I am looking for an evernote client that can be used on linux and it is a pity that with quentier I cannot complete the download of the notes in the database.
I hope you can complete the fixes you are implementing in the future.
I will surely monitor the project. Keep up with the good work
